packages/base-test-utils/src/index.ts

```diff
@@ -0,0 +1,64 @@
+import type { CID } from 'multiformats/cid'
+import { randomCID, StreamID } from '@ceramicnetwork/streamid'
+
+export class BaseTestUtils {
```

Comment:
Could this just be a type we export from the StreamIDs package instead of adding a whole new package for this?

Reply:
This one could possibly just be in streamid's package. It's more utility code than testing code, so if we're good having it used by people using our libraries, I'd be glad to get rid of this package.